README.md

```diff
@@ -4,6 +4,16 @@
 [![plan-examples](https://github.com/aws-ia/terraform-aws-eks-blueprints/actions/workflows/plan-examples.yml/badge.svg)](https://github.com/aws-ia/terraform-aws-eks-blueprints/actions/workflows/plan-examples.yml)
 [![pre-commit](https://github.com/aws-ia/terraform-aws-eks-blueprints/actions/workflows/pre-commit.yaml/badge.svg)](https://github.com/aws-ia/terraform-aws-eks-blueprints/actions/workflows/pre-commit.yaml)
 
+___
+
+# :bangbang: Notice of Potential Breaking Changes in Version 5 :bangbang:
+The direction for EKS Blueprints in v5 will shift from providing an all-encompassing, monolithic "framework" and instead focus more on how users can organize a set of modular components to create the desired solution on Amazon EKS. 
+
+An issue is being created to provide community notice and to help track the work and cutover to v5 of EKS Blueprints.
```

Comment:
replace `cutover` with something more positive 😄. ..To track progress, and learn what's new and how the migration path would look like to upgrade your current Terraform deployments.